<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>frmConfig</class>
 <widget class="QDialog" name="frmConfig">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>320</width>
    <height>300</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>320</width>
    <height>300</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>320</width>
    <height>300</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Config</string>
  </property>
  <layout class="QVBoxLayout" name="vlConfig">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="configWidget" native="true">
     <layout class="QVBoxLayout" name="configWidgetLayout">
      <item>
       <widget class="KPTLabel" name="labStyle">
        <property name="text">
         <string>Select Style</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="cbStyle">
        <item>
         <property name="text">
          <string>System</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="cbSMStyle">
        <property name="text">
         <string>Don't use modern SM Style</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="Line" name="lineUStyle">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="labLang">
        <property name="text">
         <string>Select Language</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="cbLanguage">
        <item>
         <property name="text">
          <string>System (System)</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="Line" name="lineULang">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="labLocalMode">
        <property name="text">
         <string>Local Mode (immediately)</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="hlLocalMode">
        <item>
         <widget class="QPushButton" name="cmdChangeAdminPassword">
          <property name="minimumSize">
           <size>
            <width>75</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>Change admin password</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="hsLocalMode">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="vsConfig">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QHBoxLayout" name="hlButtons">
        <item>
         <spacer name="hsButtonSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="cmdOK">
          <property name="minimumSize">
           <size>
            <width>75</width>
            <height>25</height>
           </size>
          </property>
          <property name="text">
           <string>OK</string>
          </property>
          <property name="icon">
           <iconset theme="dialog-ok">
            <normaloff>.</normaloff>.</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="cmdCancel">
          <property name="minimumSize">
           <size>
            <width>75</width>
            <height>25</height>
           </size>
          </property>
          <property name="text">
           <string>Cancel</string>
          </property>
          <property name="icon">
           <iconset theme="dialog-cancel">
            <normaloff>.</normaloff>.</iconset>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KPTLabel</class>
   <extends>QLabel</extends>
   <header>KPTLabel.h</header>
   <slots>
    <signal>mouseRelease(QMouseEvent*)</signal>
    <signal>mousePress(QMouseEvent*)</signal>
    <signal>mouseMove(QMouseEvent*)</signal>
   </slots>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
